src/mocks/GroupMock.ts

```diff
@@ -1,4 +1,5 @@
 export interface Team {
+  Id: number
```

Comment:
다른 코드와 일관된 컨벤션 사용하도록 카멜케이스 쓰시는게 좋을 거 같아요.

```suggestion
  id: number
```